nextcloudappstore/scaffolding/app-templates/12/app/appinfo/changes.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<release xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/changes.xsd" version="0.0.1">
+    <changelog href="" />
+    <whatsNew lang="en">
+        <regular>
+            <item>First {{ app.name }} release!</item>
+        </regular>
+        <admin>
+            <item>requires {{ app.nextcloud_version }}</item>
```

Comment:
This is redundant with the version in info.xml and probably a bad example for new developers. Better example or simply remove it?